src/main.js

```diff
@@ -136,46 +142,70 @@ let time = startingMinutes * 60;
 
 function startTimer() {
         setInterval(updateCountdown, 1000);
-        document.getElementById('countdown').style.display = "block";
+        setStyleDisplay("block", document.getElementById('countdown'));
 }
+
 var min, sec;
+
 function updateCountdown() {
         const minutes = Math.floor(time / 60);
         let seconds = time % 60;
         seconds = seconds < 10 ? '0' + seconds : seconds;
 
         countdown.innerHTML = `${minutes}:${seconds}`;
         time--;
-        if (minutes == 0 && seconds == 0) {
+
+        if (minutes === 0 && seconds === 0) {
```

Comment:
This caused a problem, 'seconds' is a string and you cannot compare it like this. It ran into a never-ending negative timeout.

Reply:
true, my false. I'll push changes soon

Reply:
I've already corrected it.